### PR TITLE
feat(dial): press 0 for +

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default class StatelessDialPad extends React.Component {
         this._onDialButtonClick = this._onDialButtonClick.bind(this);
         this._onGoToCall = this._onGoToCall.bind(this);
         this._onInputChange = this._onInputChange.bind(this);
-        this._renderDialButton = this._renderDialButton.bind(this);
+        this._onReplaceLastChar = this._onReplaceLastChar.bind(this);
```
Comment (paweldomas) [inline spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js]: why `_renderDialButton` bind was removed? it's used hasn't changed

Comment (paweldomas) [reply spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js]: was it not necessary? Im too lazy to read the code

Comment (virtuacoplenny) [reply spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js]: Correct, the binding is not necessary. I didn't realize I left this change in though and it'll be better if I revert the change to keep the PR a bit more pure.

**Comment by virtuacoplenny** [comment]
Screen Shot 2019-09-11 at 9 43 29 AM

**Pull request merged**